vllm/engine/metrics.py

```diff
@@ -117,6 +117,16 @@ def __init__(self, labelnames: List[str], max_model_len: int):
             documentation="Average generation throughput in tokens/s.",
             labelnames=labelnames,
         )
+        # Deprecated in favor of vllm:request_prompt_tokens_sum
+        self.counter_prompt_tokens = Counter(
+            name="vllm:prompt_tokens_total",
+            documentation="Number of prefill tokens processed.",
+            labelnames=labelnames)
+        # Deprecated in favor of vllm:request_generation_tokens_sum
+        self.counter_generation_tokens = Counter(
+            name="vllm:generation_tokens_total",
+            documentation="Number of generation tokens processed.",
+            labelnames=labelnames)
```

Comment:
With my suggestion for how to properly compute the request metric summaries, these will no longer be deprecated

Reply:
( since these will compute the "realtime" throughput )
( and the others will compute the histogram of generated tokens / histogram of prefill tokens for completed requests )